scripts/lint

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+cd $(dirname $0)/..
+
+golangci-lint run -D megacheck -E unused,gosimple,staticcheck
```

Comment:
I think this should be an edit to `scripts/validate` (which currently lints using `golint`), but as a separate PR.

Reply:
Removed the lint script. Agree, we should do it in separate PR.